ecl/hql/hqlmanifest.cpp

```diff
@@ -198,6 +198,31 @@ void ResourceManifest::loadResource(const char *filepath, MemoryBuffer &content)
     read(fio, 0, (size32_t) f->size(), content);
 }
 
+void queryJfrog(const char *localpath, IPropertyTree &item)
```

Comment:
style: We use query as a prefix for functions that return a value, e.g. queryName().

Reply:
Changed to callJfrog.